### PR TITLE
[EPAD8-1300] USWDS Update: Sitewide Alert

### PR DESCRIPTION
**Pull request by kmonahan**

### CODE REVIEW

File: services/drupal/web/themes/epa_theme/source/_patterns/03-uswds/site-alert/_site-alert.scss [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  .usa-alert__text {
+    > :last-child {
+      margin-bottom: 0;
+    }
+  }
+
+  .usa-alert__body {
+    display: flex;
+    justify-content: space-between;
+  }
```
Comment (dcmouyard) [inline services/drupal/web/themes/epa_theme/source/_patterns/03-uswds/site-alert/_site-alert.scss:18]: @kmonahan What is this used for? It currently puts the title and content on the same line.

uswds-site-alert

Comment (kmonahan) [reply services/drupal/web/themes/epa_theme/source/_patterns/03-uswds/site-alert/_site-alert.scss:18]: The Close button

Comment (kmonahan) [reply services/drupal/web/themes/epa_theme/source/_patterns/03-uswds/site-alert/_site-alert.scss:18]: It's really just changing the class selector on https://github.com/USEPA/webcms/blob/main/services/drupal/web/themes/epa_theme/source/_patterns/05-components/sitewide-alert/_sitewide-alert.scss#L16-L19

Comment (dcmouyard) [reply services/drupal/web/themes/epa_theme/source/_patterns/03-uswds/site-alert/_site-alert.scss:18]: Ah, is this for the close button on site alerts that can be closed?

Comment (kmonahan) [reply services/drupal/web/themes/epa_theme/source/_patterns/03-uswds/site-alert/_site-alert.scss:18]: Yep, exactly.

**Pull request merged**